codebuild/samples/linux-smoke-tests.yml

```diff
@@ -9,6 +9,7 @@ phases:
       - echo Build started on `date`
       - $CODEBUILD_SRC_DIR/codebuild/samples/setup-linux.sh
       - $CODEBUILD_SRC_DIR/codebuild/samples/connect-linux.sh
+      - $CODEBUILD_SRC_DIR/codebuild/samples/connect-auth-linux.sh
```

Comment:
Not familiar with this. Is there anything this could be confused with and if so, should we explicitly name this connect-custom-auth-linux?

Reply:
Possibly? The Codebuild shell script is just a set of commands that tells the Codebuld instance how to compile/run the sample and what arguments to pass. Some of them do double duty (like `connect-linux.sh`) but some are just a single command.

I'll ask in the team meeting 👍